po/fr.po

```diff
@@ -0,0 +1,354 @@
+msgstr "Ajout de la prise en charge des lecteurs vidéo personnalisés"
+
+#: data/de.k_bo.Televido.metainfo.xml.in:72
+msgid "Added support Daikhan as a video player"
+msgstr "Ajout de la prise en charge de Daikhan comme lecteur vidéo"
+
+#: data/de.k_bo.Televido.metainfo.xml.in:78
+msgid "If an error occurs, a graphical error message is shown."
+msgstr "Si une erreur se produit, un message sera affiché."
+
+#: data/de.k_bo.Televido.metainfo.xml.in:83
+msgid "Initital release"
+msgstr "Première version"
+
+#: src/application.rs:129
+msgid "Failed to play video stream"
+msgstr "Échec de la lecture du flux vidéo"
+
+#: src/application.rs:172
+msgid "Failed to launch video downloader"
+msgstr "Échec du lancement du lecteur vidéo"
+
+#: src/help-overlay.blp:10
+msgid "General"
+msgstr "Général"
+
+#: src/help-overlay.blp:13
+msgid "Show livestreams"
+msgstr "Afficher les directs"
+
+#: src/help-overlay.blp:18
+msgid "Show mediathek"
+msgstr "Afficher la médiathèque"
+
+#: src/help-overlay.blp:23 src/window.blp:41
+msgid "Reload"
+msgstr "Recharger"
+
+#: src/help-overlay.blp:28
+msgid "Quit application"
+msgstr "Quitter l’application"
+
+#: src/launcher/selector.blp:18
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/launcher/selector.blp:24
+msgid "Confirm"
+msgstr "Confirmer"
+
+#: src/launcher/selector.blp:38
+msgid "Custom program"
+msgstr "Programme personnalisé"
+
+#. translators: `{}` is replaced by the application ID, e.g. `org.example.Application`
+#: src/launcher/selector.rs:115
+msgid "Could not find application “{}”"
+msgstr "Impossible de trouver l’application « {} »"
+
+#: src/launcher/selector.rs:169
+msgid "Failed to load external applications"
+msgstr "Échec du chargement des applications externes"
+
+#: src/launcher/selector.rs:173 src/utils.rs:183
+msgid "See the terminal output for details."
+msgstr "Regardez la sortie du terminal pour plus de détails."
+
+#: src/launcher/selector.rs:280
+msgid "Select video player"
+msgstr "Sélectionnez un lecteur vidéo"
+
+#: src/launcher/selector.rs:281
+msgid "Select one of the following external programs to stream content."
+msgstr ""
+"Sélectionnez l’un de ces programmes externes pour suivre la diffusion d’un "
+"contenu."
+
+#: src/launcher/selector.rs:284
+msgid "Select video downloader"
+msgstr "Sélectionnez un programme de téléchargement"
+
+#: src/launcher/selector.rs:285
+msgid "Select one of the following external programs to download content."
+msgstr ""
+"Sélectionnez l’un de ces programmes externes pour télécharger un contenu."
+
+#: src/launcher/selector.rs:292
+msgid ""
+"You can also specify a custom application ID (e.g. org.example.Application) "
+"of a different program that supports DBus activation, is able to open a "
+"https:// URI and is accessible from the context of this application."
+msgstr ""
+"Vous pouvez également spécifier un identifiant d’application personnalisé "
+"(ex.: org.exemple.Application) pour un programme différent qui prend en "
```

Comment:
The application ID shouldn't be translated because example.org is one of the few [reserved example domain names](https://datatracker.ietf.org/doc/html/rfc2606#section-3).

Reply:
Corrected!